### PR TITLE
Sensors to Main branch.

### PR DESCRIPTION
**Pull request by NielsGoBrrr**
Sensors branch should be fully done, file changes should be closely inspected, and some debug statements still remain.
(sensor_data table & print with log function, amongst others.)

### CODE REVIEW

File: main/wifi/wifi.c [hunk 1]
```diff
@@ -116,19 +116,24 @@ esp_err_t wifi_send_data_to_server(
 		.event_handler = http_event_handle,
 		.crt_bundle_attach = esp_crt_bundle_attach,
 	};
+	LOG("Init begin");
 	esp_http_client_handle_t client = esp_http_client_init(&config);
+	LOG("EEEEEEEEE");
```
Comment (jaschutte) [inline main/wifi/wifi.c:121]: Rename this something

File: main/sensors/sensors.c [hunk 2]
```diff
@@ -0,0 +1,93 @@
+#include "sensors.h"
+#include "adc.h"
+
+static uint64_t count = 0;
```
Comment (jaschutte) [inline main/sensors/sensors.c:4]: Could you rename this to something more clear? soil_capacity_timer_count or something?

**Review by jaschutte — CHANGES_REQUESTED** [review]

File: main/sensors/sensors.c (hunk 2 above)
Comment (NielsGoBrrr) [reply main/sensors/sensors.c:4]: Will def break it but alrighty

Comment (NielsGoBrrr) [reply main/sensors/sensors.c:4]: Probably not actually.

**Review by jaschutte — APPROVED** [review]

**Pull request merged**